docker/app/Dockerfile

```diff
@@ -10,16 +10,15 @@ RUN npm config set unsafe-perm true && npm install -g npm@7.6.3
 RUN mkdir -p /data
 WORKDIR /data
 
-# copy npm cache directory; as optimization for npm install
-COPY --from=sillsdev/web-languageforge:npm-cache /root/.npm /root/.npm
-
 # Copy in files needed for compilation, located in the repo root
 COPY typings ./typings/
-COPY package.json package-lock.json webpack.config.js webpack-dev.config.js webpack-prd.config.js tsconfig.json tslint.json ./
+COPY package.json webpack.config.js webpack-dev.config.js webpack-prd.config.js tsconfig.json tslint.json ./
+
+# Use built node_modules from host
+COPY node_modules ./node_modules/
```

Comment:
Because this is a multi-stage build, the `node_modules` folder is actually not included in the final app image build.  This line is part of the ui-builder stage which is discarded.  Only the `dist` folder resulting from ui-builder is persisted in the app image.